packages/palette-docs/package.json

```diff
@@ -79,6 +79,7 @@
     "docz-utils": "^0.13.6",
     "gatsby": "2.3.34",
     "gatsby-image": "^2.0.20",
+    "gatsby-link": "^2.4.13",
```

Comment:
See this [recent Gatsby issue](https://github.com/gatsbyjs/gatsby/issues/25708#issuecomment-658110313) /shrug